lightningd/options.c

```diff
@@ -610,16 +610,27 @@ void setup_color_and_alias(struct lightningd *ld)
 
 	if (!ld->alias) {
 		u64 adjective, noun;
+		char *name;
+		int taillen;
 
 		memcpy(&adjective, der+3, sizeof(adjective));
 		memcpy(&noun, der+3+sizeof(adjective), sizeof(noun));
 		noun %= ARRAY_SIZE(codename_noun);
 		adjective %= ARRAY_SIZE(codename_adjective);
+
+		/* Only use 32 characters */
+		name = tal_fmt(ld, "%s%s-",
```

Comment:
Suggest doing this only if DEVELOPER=1.

Reply:
You might be right; wasn't sure about this one.  Will change.